README.md

```diff
@@ -36,7 +36,7 @@ The Infusion library is the cornerstone of a number of projects. You can see som
 
 ## How Do I Create an Infusion Package? ##
 
-Strictly speaking, Infusion can be used directly from source (i.e. by including each individual required file). However, for simplicity and performance reasons, you may wish to create a concatenated, minified file. The Grunt build options described below will also allow you to remove any unneeded features or libraries that you may already have in your project.
+Strictly speaking, Infusion can be used directly from source (i.e. by including each individual required file). However, for simplicity and performance reasons, you may wish to create a concatenated, minified file. Source maps for the minified file are automatically generated to make debugging easier. The Grunt build options described below will allow you to remove any unneeded features or libraries that you may already have in your project.
```

Comment:
Could you add a little more to give the user some idea how to use source maps? Perhaps i) some pointers to the relevant docs for at least FF, Chrome - and, presumably, the user has to arrange for the built file to be hosted from the place it was generated by the build in a statically hosted image of the whole project?